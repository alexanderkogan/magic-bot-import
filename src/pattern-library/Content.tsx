import * as React from 'react'
import { Props } from '../react-typings'

export default (props: Props) => <div className="obc_content-container">{props.children}</div>
