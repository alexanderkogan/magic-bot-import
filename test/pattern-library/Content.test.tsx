import * as React from 'react'

import Content from '../../src/pattern-library/Content'
import { mount } from '../test-utils'

describe('Content', () => {
    it('renders', () => {
        const element = mount(<Content>children</Content>)
        expect(element).toMatchSnapshot()
    })
})
