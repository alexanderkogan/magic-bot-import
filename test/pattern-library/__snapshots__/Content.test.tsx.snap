// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content renders 1`] = `
<Component>
  <div
    className="obc_content-container"
  >
    children
  </div>
</Component>
`;
